recipes/libavif/all/conanfile.py

```diff
@@ -0,0 +1,123 @@
+import functools
+import os
+import textwrap
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.43.0"
+
+codecs = {
+    "aom": "libaom-av1/3.1.2",
```

Comment:
The current libaom release is 3.2.0. Is libaom 3.2.0 available in Conan?

Similarly for dav1d. The current dav1d release is 0.9.2.

Reply:
Those are the latest versions currently packaged in CCI.